apps/cowswap-frontend/src/legacy/state/swap/reducer.ts

```diff
@@ -78,11 +78,19 @@ export default createReducer<SwapState>(initialState, (builder) =>
       const { chainId, recipient, inputCurrencyId, outputCurrencyId, inputCurrencyAmount, outputCurrencyAmount } =
         payload
 
-      const typedValue = state.independentField === Field.INPUT ? inputCurrencyAmount : outputCurrencyAmount
+      const probableTypedValue = state.independentField === Field.INPUT ? inputCurrencyAmount : outputCurrencyAmount
```

Comment:
i find strange the "probable" word here, but not sure what name is better. Maybe `typedValueForNormalFlows`

i also find strange the word "typedValue". Anyways, not too imporant

Reply:
Renamed to `defaultTypedValue`